README.md

```diff
@@ -1,8 +1,6 @@
 ![Spectral logo](img/spectral-banner.png)
 
-[![Test Coverage](https://api.codeclimate.com/v1/badges/1aa53502913a428f40ac/test_coverage)](https://codeclimate.com/github/stoplightio/spectral/test_coverage)
-[![Maintainability](https://api.codeclimate.com/v1/badges/1aa53502913a428f40ac/maintainability)](https://codeclimate.com/github/stoplightio/spectral/maintainability)
-[![Build Status](https://dev.azure.com/vncz/vncz/_apis/build/status/stoplightio.spectral?branchName=develop)](https://dev.azure.com/vncz/vncz/_build/latest?definitionId=4&branchName=develop)
```

Comment:
@philsturgeon Could we keep the Windows build one?

Reply:
I'd like to bring it back but I don't think we want the vncz personal project badge anymore. We should A) get windows building into Circle and drop Azure, or B) get a stoplightio org for Azure.

Reply:
:+1: